src/components/ChartContainer/D3Container.js

```diff
@@ -254,77 +177,47 @@ function D3Container({ dashboardState, dashboardActions, store }) {
           filterDisabled={filterDisabled}
         />
       </Grid>
-      <Tabs
-        value={tabValue}
-        onChange={(event, index) => handleTabChange(event, index)}
-        indicatorColor="primary"
-      >
-        <Tab label="All Measures" className="d3-container__tab-button" />
-        <Tab label="Individual Measures" className="d3-container__tab-button" />
-      </Tabs>
-      <TabPanel value={tabValue} index={1}>
-        <Grid container className="d3-container__chart-holder">
-          <Grid item sx={{ width: '25%' }}>
-            <MeasureSelector
-              currentResults={store.currentResults}
-              measure={byLineMeasure.measure}
-              handleMeasureChange={handleByLineChange}
-            />
-          </Grid>
-          <Grid item>
-            <D3Chart
-              displayData={byLineDisplayData}
-              graphWidth={graphWidth}
-              colorMapping={byLineColorMap}
-              measureInfo={store.info}
-              currentTimeline={currentTimeline}
-            />
-          </Grid>
-        </Grid>
-        <DisplayTable
-          rowData={MeasureTable.formatData(byLineCurrentResults)}
-          headerInfo={MeasureTable.subHeaderInfo}
-          pageSize={MeasureTable.pageSize}
-          useCheckBox
-          handleCheckBoxEvent={handleByLineMeasureChange}
-          selectedRows={byLineSelectedMeasures}
-          colorMapping={byLineColorMap}
+      <Grid className="d3-container__main-chart">
+        <D3Chart
+          displayData={displayData}
+          colorMapping={colorMap}
+          measureInfo={store.info}
+          graphWidth={graphWidth}
+          currentTimeline={currentTimeline}
         />
-      </TabPanel>
-      <TabPanel value={tabValue} index={0}>
-        <Grid container className="d3-container__chart-holder">
-          <Grid item className="d3-container__main-chart">
-            <D3Chart
-              displayData={displayData}
-              colorMapping={colorMap}
-              measureInfo={store.info}
-              graphWidth={graphWidth}
-              currentTimeline={currentTimeline}
-            />
-          </Grid>
+      </Grid>
+      <Grid className="d3-container__bottom-display">
+        <Grid className="d3-container__measure-selector">
+          <Typography className="d3-container__selector-title">Detailed View: </Typography>
+          <MeasureSelector
```

Comment:
MeasureSelector is missing `measure={activeMeasure}` to set the selected value.

Reply:
You know what, I'll readd it. I removed it because it wasn't going to matter since it'll be hidden by the transition anyway.